ppb/systems.py

```diff
@@ -40,7 +57,61 @@ def on_update(self, update, signal):
         for pygame_event in pygame.event.get():
             ppb_event = self.event_map.get(pygame_event.type)
             if ppb_event is not None:
-                signal(ppb_event())
+                signal(getattr(self, ppb_event)(pygame_event, update.scene))
+
+    def quit(self, event, scene):
+        return events.Quit()
+
+    def mouse_motion(self, event, scene):
+        screen_position = Vector(*event.pos)
+        camera = list(scene.get(tag="main_camera"))[0]
+        game_position = camera.translate_to_frame(screen_position)
+        delta = Vector(*event.rel) * (1/camera.pixel_ratio)
```

Comment:
Would be a good place for a comment about what the `delta` is

Reply:
The big thing I need to figure out is if delta or "rel" is more understandable.